test/lint.js

```diff
@@ -84,7 +84,7 @@ function load(...files) {
         try {
           if (file.indexOf('browsers' + path.sep) !== -1) {
             hasSchemaErrors = testSchema(file, './../../schemas/browsers.schema.json');
-            hasStyleErrors = testLinks(file);
+            hasLinkErrors = testLinks(file);
```

Comment:
This&nbsp;is&nbsp;a&nbsp;typo from&nbsp;#4847.

Reply:
This change should still go into its own PR.  :)